HeroLib/Class/Unit/Main.lua

```diff
@@ -15,7 +15,34 @@ local Item = HL.Item
 local tonumber = tonumber
 -- File Locals
 
-
+-- Blizzard API Mappings
```

Comment:
Better do that in a sub scope with a do..end block. There is a limit of 200 local variables per file, and doing so make it slightly faster aswell.
Although, we can still split files if we need to.